taskt/Core/ImageRecognitionFingerPrint.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Drawing;
+
+namespace taskt.Core
+{
+    /// <summary>
+    /// Used by Image Recognition to generate and verify a unique fingerprint of an image
+    /// </summary>
+    public class ImageRecognitionFingerPrint
+    {
+        public int PixelId { get; set; }
```

Comment:
I feel like this file might be in the wrong place. It feels kind of random for the Core folder. What do you think?

Reply:
I know, let's discuss its placement in Standup.

Reply:
Moved to Core/Utilities/CommandUtilities Directory